modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -776,6 +777,10 @@ private void _addOrUpdateNestedObjectEntries(
 						objectDefinition, objectRelationship, primaryKey,
 						nestedObjectEntry.getId());
 				}
+
+				if (!nestedObjectEntries.isEmpty()) {
```

Comment:
I would like to test the following use case:
- Try to create (POST) a University with `"universityStudents": []`

being:
- University, Student: Custom Objects
- universityStudents: 1 to M relationship

Check if the response contains the `universityStudents` field

Reply:
In that case the response currently won't contain the nested field. Is that ok? I think it is because sending [] is equivalent to sending nothing.

Reply:
See https://github.com/liferay-headless/liferay-portal/pull/1224/commits/96065de605ee44f121b1aa8b7c046c555415ca03.

Reply:
Based on the current behaviour, I think the empty list should be returned:

`curl -X GET -u 'test@liferay.com:test' -H 'Content-Type: application/json' 'http://localhost:8080/o/c/students/47289?nestedFields=studentsSubjects&restrictFields=actions'`

```
{
  "creator" : {
    "additionalName" : "",
    "contentType" : "UserAccount",
    "familyName" : "Test",
    "givenName" : "Test",
    "id" : 20123,
    "name" : "Test Test"
  },
  "dateCreated" : "2023-05-09T15:10:37Z",
  "dateModified" : "2023-05-09T15:10:37Z",
  "externalReferenceCode" : "c9c3ddcb-867e-e8fc-827e-8ddbe26428b6",
  "id" : 47289,
  "keywords" : [ ],
  "status" : {
    "code" : 0,
    "label" : "approved",
    "label_i18n" : "Approved"
  },
  "taxonomyCategoryBriefs" : [ ],
  "studentsSubjects" : [ ],
  "studentName" : "student2"
}
```